pom.xml

```diff
@@ -165,8 +165,7 @@
                     </systemPropertyVariables>
                     <properties>
                         <configurationParameters>junit.jupiter.execution.parallel.enabled = true
-                            <!-- Execute top-level classes in sequentially but their methods in parallel -->junit.jupiter.execution.parallel.mode.default = concurrent
-                            junit.jupiter.execution.parallel.mode.classes.default = same_thread
+                            <!-- Execute top-level classes in sequentially but their methods in parallel -->junit.jupiter.execution.parallel.mode.default = concurrent junit.jupiter.execution.parallel.mode.classes.default = same_thread
```

Comment:
This part looks strange, pls update it to something like this:

```
                        <!-- Execute top-level classes in sequentially but their methods in parallel -->
                        <configurationParameters>
                            junit.jupiter.execution.parallel.enabled = true
                            junit.jupiter.execution.parallel.mode.default = concurrent
                            junit.jupiter.execution.parallel.mode.classes.default = same_thread
                        </configurationParameters>
```

Reply:
it wasn't me, formatter did that. I can only presume that it happens to everyone who runs `mvn clean verify`? No reason why my formatter should be specific, anyway, I'll try your suggestion.

Reply:
formatter keeps pushing it to one line, but I'll move at least comment